QA/PoroDevTest/src/main/java/common/ui_setup/pom_setup/BasePage.java

```diff
@@ -1,16 +1,59 @@
 package common.ui_setup.pom_setup;
 
-import common.ui_setup.ElementControl;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
 
 public abstract class BasePage {
-    public ElementControl elementControl;
     protected WebDriver driver;
 
     public BasePage(WebDriver driver) {
         this.driver = driver;
-        elementControl = new ElementControl(driver);
         PageFactory.initElements(driver, this);
     }
-}
+
+    public static void clickElement(WebElement element) {
```

Comment:
Overkill. You don't need help methods where you use only one method, which is .click() method in this case.

Reply:
I have wrapper methods like that because if some of that element control methods get deprecated I can easily go and reformat just one method, not everywhere where I used element.click();.
That make sense to me.